backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/report/AD_Process_StepDef.java

```diff
@@ -0,0 +1,134 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.cucumber.stepdefs.report;
+
+import de.metas.adempiere.model.I_C_Invoice;
+import de.metas.adempiere.model.I_C_Order;
+import de.metas.cucumber.stepdefs.C_Order_StepDefData;
+import de.metas.cucumber.stepdefs.DataTableRow;
+import de.metas.cucumber.stepdefs.DataTableRows;
+import de.metas.cucumber.stepdefs.StepDefDataIdentifier;
+import de.metas.cucumber.stepdefs.invoice.C_Invoice_StepDefData;
+import de.metas.cucumber.stepdefs.shipment.M_InOut_StepDefData;
+import de.metas.i18n.Language;
+import de.metas.process.AdProcessId;
+import de.metas.process.IADPInstanceDAO;
+import de.metas.process.IADProcessDAO;
+import de.metas.process.ProcessCalledFrom;
+import de.metas.process.ProcessInfo;
+import de.metas.report.server.LocalReportServer;
+import de.metas.report.server.OutputType;
+import de.metas.report.server.ReportResult;
+import de.metas.util.Services;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.And;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.adempiere.ad.trx.api.ITrx;
+import org.adempiere.archive.api.ArchiveRequest;
+import org.adempiere.archive.api.IArchiveBL;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.util.lang.impl.TableRecordReference;
+import org.compiere.model.I_AD_Process;
+import org.compiere.model.I_AD_Table;
+import org.compiere.model.I_M_InOut;
+import org.compiere.util.Env;
+import org.springframework.core.io.ByteArrayResource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RequiredArgsConstructor
+public class AD_Process_StepDef
+{
+	@NonNull private final C_Order_StepDefData orderTable;
+	@NonNull private final M_InOut_StepDefData inOutTable;
+	@NonNull private final C_Invoice_StepDefData invoiceLineTable;
+
+	@NonNull private final IADProcessDAO processDAO = Services.get(IADProcessDAO.class);
+	@NonNull private final IADPInstanceDAO pInstanceDAO = Services.get(IADPInstanceDAO.class);
+	@NonNull private final IArchiveBL archiveBL = Services.get(IArchiveBL.class);
+
+	@And("The jasper process is run")
+	public void run_jasper_processes(@NonNull final DataTable dataTable)
+	{
+		DataTableRows.of(dataTable).forEach(this::run_jasper_process);
+	}
+
+	public void run_jasper_process(@NonNull final DataTableRow row)
+	{
+		final String value = row.getAsString(I_AD_Process.COLUMNNAME_Value);
+		final AdProcessId processId = processDAO.retrieveProcessIdByValue(value);
+		assertThat(processId).isNotNull();
+
+		final String tableName = row.getAsString(I_AD_Table.COLUMNNAME_TableName);
```

Comment:
IMHO you don't need "TableName" column in cucumber feature file. See de.metas.cucumber.stepdefs.accounting.IdentifiersResolver#getTableRecordReference.
Maybe we shall move it out of accounting package and make it as general use.